cg/cli/workflow/taxprofiler/base.py

```diff
@@ -68,10 +75,13 @@ def config_case(
 @OPTION_WORKDIR
 @OPTION_FROM_START
 @OPTION_PROFILE
+@OPTION_TOWER
```

Comment:
let's remove this option. It's confusing and when we are using nextflow we do not want to use tower